plugin-ci/orb.yml

```diff
@@ -104,11 +104,22 @@ jobs:
       - checkout
       - *restore_cache
       - run: make dist
+      - run:
+          name: "Generating release notes"
+          command: |
+            echo "### Supported Mattermost Server Versions: $(cat plugin.json | jq .min_server_version -r)+"  >> release-notes.md
+            if [[ $(git tag -l | wc -l) -eq 1 ]]; then
+              git log --pretty=oneline --abbrev-commit --no-decorate --no-color $(git rev-list --max-parents=0 HEAD) HEAD >> release-notes.md;
+            else
+              git log --pretty=oneline --abbrev-commit --no-decorate --no-color tags/$(git describe --tags --abbrev=0)^..HEAD >> release-notes.md;
```

Comment:
The current format has a bullet point in front of every commit. I would be nice, if this could also be done via automation. Not blocking for this PR

Reply:
I can make the change.

Reply:
Sorry, I don't see where is in done in the code. Would you please point it out for me?

Reply:
Just made the commit now.